src/Authentication/Login.php

```diff
@@ -106,6 +92,7 @@ function () use ($client, $loginRequest, $sub) {
                 $client->getClient()->getEmitter()->detach($sub);
                 $rawResponse = $client->getClient()->send($client->getGuzzleRequest($loginRequest));
                 if ($this->format == 'json') {
+                    /** @var array|object|mixed $response */
```

Comment:
ještě tam chybí bool|int|string  :-D

Reply:
v tomto kontexte mi to tak dava zmysel, ... jedna sa o vyskedok json_decode, ... ktory je bud array,object alebo skalarna hodnota (ktora sa v kode nizsie konvertuje).

Reply:
to měl být vtip :) :) jinak to je v pohodě :)

Reply:
ok :) ako stacilo by tam `mixed`, tak som nevedel presne ako to myslis